octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -496,6 +497,40 @@ def default_laserhead_max_temperature(self):
 
         return LASERHEAD_MAX_TEMP_FALLBACK
 
+    @property
+    def current_laserhead_high_temperature_warn_offset(self):
+        """Return the current laser head high temperature warn offset.
+
+		Returns:
+			float: Laser head high tmp warn offset
+		"""
+        current_laserhead_properties = self._get_laserhead_properties()
+
+        # Handle the exceptions
+        if ((isinstance(current_laserhead_properties, dict) is False) or
+                ("high_temperature_offset" not in current_laserhead_properties) or
+                (isinstance(current_laserhead_properties["high_temperature_offset"], float) is False)):
+            # Apply fallback
+            self._logger.debug("Current laserhead properties: {}".format(current_laserhead_properties))
+            self._logger.exception(
+                "Current Laserhead high temperature warn offset couldn't be retrieved, fallback to the temperature value of: {}".format(
+                    self.default_laserhead_high_temperature_warn_offset))
+            return self.default_laserhead_high_temperature_warn_offset
```

Comment:
I think this is missing the parenthesis. On the other hand, are we planning to add some more logic to `default_laserhead_high_temperature_warn_offset`? Or should we just use the const?

Reply:
it is a property thats why we don't need the parantheses. Currently we only use the const, I reused the way that ahmed implemented for the values from the start of the laserhead profile.

Reply:
Aaah you're right, I didn't notice about the property :)